server/sonar-server/src/main/java/org/sonar/server/ws/DeprecatedRestWebServiceFilter.java

```diff
@@ -44,12 +51,15 @@
 import static org.sonarqube.ws.client.setting.SettingsWsParameters.PARAM_KEY;
 import static org.sonarqube.ws.client.setting.SettingsWsParameters.PARAM_KEYS;
 import static org.sonarqube.ws.client.setting.SettingsWsParameters.PARAM_VALUE;
+import static org.sonarqube.ws.client.setting.SettingsWsParameters.PARAM_VALUES;
 
 /**
  * This filter is used to execute some deprecated Java WS, that were using REST
  */
 public class DeprecatedRestWebServiceFilter extends ServletFilter {
 
+  private final static Splitter VALUE_SPLITTER = Splitter.on(",").omitEmptyStrings();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AModifiersOrderCheck)